app/css/ViewEvaluation.css

```diff
@@ -1,6 +1,6 @@
 body {
   font-family: Arial, sans-serif;
-  background-color: burlywood; /* Updated */

```

Comment:
two lines of code for the background color when only one is needed. in other words, it is better we remove this line of code as it serves no purpose